src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,49 @@
+.stopwatch {
+  position: relative;
+  border: 1px solid #000;
+  border-radius: 50%;
+  @include size($watch-height);
+
+  &__min {
+    @extend %position;
+    height: $min-height;
+    width: 4px;
+    border-radius: 4px;
+    background-color: #6c00b1;
+    z-index: 2;
+    animation: rotate 3600s linear infinite;
+  }
+
+  &__sec {
+    @extend %position;
+    height: $sec-height;
+    width: 4px;
+    border-radius: 4px;
+    background-color: #39a7ff;
+    z-index: 1;
+    animation: rotate 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    height: 20px;
+    width: 20px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-40%, -50%);
+    background-color: #000;
+    border-radius: 50%;
+    z-index: 3;
+  }
+
+  @keyframes rotate {
+    0% {
+      transform: rotate(0deg);
+      transform-origin: bottom;
+    }
+    100% {
+      transform: rotate(360deg);
+      transform-origin: bottom;
```

Comment:
transform-origin: bottom;
does not change why not to set it by default?

Reply:
transform origin is by default center. when it is in the center line turns from the middle not from the end of it.